.github/workflows/mlmodeldownloader.yml

```diff
@@ -52,8 +52,7 @@ jobs:
     runs-on: macOS-latest
```

Comment:
Temporarily comment line 51 to test the `spm-cron` job before tonight.

Reply:
Done!

Reply:
LGTM!